extensions/jackson/runtime/src/main/java/io/quarkus/jackson/runtime/JacksonBuildTimeConfig.java

```diff
@@ -17,6 +17,13 @@
     @ConfigItem(defaultValue = "false")
     public boolean failOnUnknownProperties;
 
+    /**
+     * If enabled, Jackson will fail when no accessors are found for a type.
+     * This is disabled by default to match the default Jackson behavior.
```

Comment:
You say it's disabled by default but the default value is true?

Reply:
Well, spotted. Thanks!